Pipfile

```diff
@@ -18,7 +18,7 @@ ipython = "*"
 python_version = "3.6"
 
 [scripts]
-test = "python -m unittest discover mcm/test test_*.py"
+test = "python -m unittest discover --start-directory ./mcm/test --pattern test_*.py"
```

Comment:
what does this do? how is it different than what we were doing before?

Reply:
Even though it looks different from before, it's not functionally different -- just more explicit.